backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -652,6 +660,9 @@ public static Bundle createFhirBundle(
                     .setFullUrl(pair.getFirst())
                     .setResource(pair.getSecond())));
 
+    bundle.setMeta(
+        new Meta()
+            .addTag(PROCESSING_ID_SYSTEM, processingId, PROCESSING_ID_DISPLAY.get(processingId)));
     return bundle;
```

Comment:
I think the preference here is to use the library's built in auto-creation when you call `get` on most things. 
```java
bundle.getMeta().addTag(PROCESSING_ID_SYSTEM, processingId, PROCESSING_ID_DISPLAY.get(processingId)));
```